tests/test_optics.py

```diff
@@ -570,13 +543,20 @@ def test_OpticalPSF_pupil_plane():
     ref_psf = galsim.OpticalPSF(lam_over_diam, obscuration=obscuration, nstruts=nstruts,
                                 oversampling=pp_oversampling, strut_angle=strut_angle,
                                 pad_factor=pp_pad_factor)
-    im = galsim.fits.read(os.path.join(imgdir, pp_file))
+    if os.path.isfile(os.path.join(imgdir, pp_file)):
+        im = galsim.fits.read(os.path.join(imgdir, pp_file))
+    else:
```

Comment:
Same comment as on the other branch.  Let's emit a warning here, so if it happens in a nosetests run, people will notice.  (Since it shouldn't.)